### ISSUE TITLE
Nov 11 - 12: Create your own blockchain network

### ISSUE THREAD

**Issue by meaghanlewis** [microsoft/ReactorPlanning#120]
# Event Information
| Category | Details |
|-----------|---------|
| Reactor Topic | Emerging Technologies |
| Workshop Topic | Blockchain |
| Workshop Duration | 55 min | 
| Title | Create Your Own Blockchain Network |
| Description | What is this session about? This is the second session in a three-part series on blockchain technology. In this session, we'll create our own blockchains using Visual Studio Code and the Blockchain Development Kit extension. We'll write, test, and deploy our contracts to a local blockchain network running on the Ethereum Virtual Machine. Who is it aimed at? This is designed for those who have a basic understanding of blockchain technology and want to get some hands-on experience. Why should you join? We will explore blockchain development together and discover tools and strategies that can be used to program applications. About the speaker Meaghan Lewis has been working in the tech industry for over a decade. She is currently a Technical Program Manager at Microsoft and focuses on creating and delivering learning experiences centered around emerging technologies such blockchain and quantum computing. Meaghan enjoys tinkering on personal projects and collaborating with other enthusiasts to explore new things. She is a California native who currently lives in the San Francisco Bay Area with her husband and two dogs. |

# Workshop(s) details
| Date |  Local Start Time |  Location |  Speaker | SharePoint ID | Platform |
|------|-------------------|-----------|----------|---------------|----------|
| 11/11 | 5pm PT | US, CAN, AUS | @meaghanlewis | 11880 | |
| 11/12 | 8:30am PT | London, Stockholm, Tel Aviv, Abu Dhabi, Bangalore | @meaghanlewis  | 11881 | |

# Event Checklist

## 10 Weeks Prior
- [x] Reactor Content Team (RCT) opens a new issue for an upcoming local workshop filling in the Event Information and Workshop details tables. RCT member who owns content (content owner) this week will be the assignee.
- [x] RCT and content owner discuss details of the event in the comments of this issue. RCT adds a _confirmed-event_ label and _local-digital-event_ tag to the issue and adds event to the __Local Digital Events__ project. Issue will appear in the __Upcoming Streams__ column
- [x] RCT adds a _ready-for-marketing-edit_ label so the RCT can review title and description

## 8 Weeks Prior
- [ ] RCT does a review of title and description and removes the _ready-for-marketing-edit_ label 
- [ ] RCT moves the issue to the __Ready for SharePoint__ column
- [ ] RCT informs the Reactor Marketing Team of the new event information
- [ ] RCT confirms the speaker
- [ ] RCT provides speaker with assets (e.g. slides, overlays) and the survey data for the event
- [ ] RCT moves the issue to the __Content Preparation In Progress__ column

## 5 weeks Prior
- [ ] Content owner opens a PR with the new content for the workshop on the [customer-facing Reactor repository](https://github.com/microsoft/Reactors/tree/main/online-event-resources) and adds the _content-development_ and _local-digital-event_ tag to the PR. 
- [ ] RCT reviews, approves, and merges the changes
- [ ] RCT shares content with the vendor/instructor
- [ ] RCT provides vendor/instructor with assets (e.g. slides, overlays) and the survey data for the event
- [ ] RCT ensures that instructors have received an invite to the workshops and that PM team works with instructors to get setup in the platform (e.g. Teams, Teams Live Event)

## 1 Week Prior
- [ ] Connect vendor/instructor with the PM(s) for the locations where this event has been marketed
- [ ] Check-in with vendor/instructor that they are prepared for the workshop
- [ ] RCT moves the issue to the __Ready to Stream__ column

## During and After Stream
- [ ] Instructor (and optional moderator) run the stream
- [ ] _(Optional)_RCT submits recording to be uploaded to the Reactor YouTube and to be made available on LearnTV
- [ ] RCT moves issue to the __Stream Complete__ column

**Comment by mebaumb** [comment]
edit/clarification on abstract:

In this workshop, we'll create our own blockchains. 

> Start off by installing the tools you need like Visual Studio Code to create smart contracts. 
_Do they need to do this before the workshop? or will you do it during the workshop?_

Then we will write, test, and deploy our contracts to a local blockchain network running on the Ethereum Virtual Machine. Join Program Manager, Meaghan Lewis, to explore blockchain development together.

**Comment by meaghanlewis** [comment]
Oh good question. I will not cover installation during the workshop so that's a prereq.

**Comment by meaghanlewis** [comment]
Maybe this edit works:
In this workshop, we'll create our own blockchains using Visual Studio Code and the Blockchain Development Kit extension. We will write, test, and deploy our contracts to a local blockchain network running on the Ethereum Virtual Machine. Join Program Manager, Meaghan Lewis, to explore blockchain development together.

What do you think?

**Comment by mebaumb** [comment]
Yeah I think that sounds great! Can you help me fill in the prereqs? 

In this workshop, we'll create our own blockchains using Visual Studio Code and the Blockchain Development Kit extension. We will write, test, and deploy our contracts to a local blockchain network running on the Ethereum Virtual Machine. Join Program Manager, Meaghan Lewis, to explore blockchain development together.

**Prerequisites:** You will need to have Visual Studio Code installed, and a basic understanding of blockchain technology to follow along. 

LMK!

**Comment by meaghanlewis** [comment]
@mebaumb made some edits inline. Feel free to go ahead and publish this one if you're happy with the description.

**Comment by mebaumb** [comment]
Event is Published!

**Comment by mebaumb** [comment]
Do you have the link to the learn content?

**Comment by meaghanlewis** [comment]
@mebaumb not for this one, no. This workshop doesn't mention that we'll be going through a Learn module, although we likely will go through some of the content that I created that will be live in a couple weeks.

**Comment by mebaumb** [comment]
Ok - Stephie is asking for linked content for each of these workshops that she can use to promote in the newsletter. Do you have anything that you can point to? or should we just leave it blank?

**Comment by meaghanlewis** [comment]
If this is needed by EOD, then no, we have nothing to link. 😞

**Comment by mebaumb** [comment]
Ok - I'll let Stephie know!

**Comment by meaghanlewis** [comment]
**What is this session about?**
This is the second session in a three-part series on bockchain. In this session, we'll create our own blockchains using Visual Studio Code and the Blockchain Development Kit extension. We'll write, test, and deploy our contracts to a local blockchain network running on the Ethereum Virtual Machine. 

**Who should attend?**
This session is targeted for those who have a basic understanding of blockchain technology, and want to get some hands-on experience.

**Why attend?**
To explore blockchain development together, discovering tools and strategies used to program applications.

**About the speaker**
Meaghan Lewis has been working in the tech industry for over a decade. She is currently a Technical Program Manager at Microsoft and focuses on creating and delivering learning experiences centered around emerging technologies like blockchain and quantum computing. Meaghan enjoys tinkering on personal projects and collaborating with other enthusiasts to explore new things. She is a California native who currently lives in the San Francisco Bay Area with her husband and two dogs.

**Comment by james-woollard** [comment]
What is this session about?
This is the second session in a three-part series on blockchain technology. In this session, we'll create our own blockchains using Visual Studio Code and the Blockchain Development Kit extension. We'll write, test, and deploy our contracts to a local blockchain network running on the Ethereum Virtual Machine.

Who is it aimed at?
This is designed for those who have a basic understanding of blockchain technology and want to get some hands-on experience.

Why should you join?
We will explore blockchain development together and discover tools and strategies that can be used to program applications.

About the speaker
Meaghan Lewis has been working in the tech industry for over a decade. She is currently a Technical Program Manager at Microsoft and focuses on creating and delivering learning experiences centered around emerging technologies such blockchain and quantum computing. Meaghan enjoys tinkering on personal projects and collaborating with other enthusiasts to explore new things. She is a California native who currently lives in the San Francisco Bay Area with her husband and two dogs.

**Comment by meaghanlewis** [comment]
@mebaumb I think this one is ready too. We can go with James' edit above.

**Comment by mebaumb** [comment]
This one has been updated!
